.gitignore

```diff
@@ -1,2 +1,4 @@
 /vendor/
-composer.lock
\ No newline at end of file
+composer.lock
+phpunit.xml
+.idea
```

Comment:
I can understand what do you want, but I think `.idea` shoud be ignored in `.gitignore_global`

Reply:
Thank you, I get it. 
So I will restore it.